docs/docsite/rst/reference_appendices/common_return_values.rst

```diff
@@ -20,56 +20,259 @@ backup_file
+                    "mode": "666",
+                    "owner": null,
+                    "regexp": null,
+                    "remote_src": null,
+                    "selevel": null,
+                    "serole": null,
+                    "setype": null,
+                    "seuser": null,
+                    "src": "/Users/foo/.ansible/tmp/ansible-tmp-1596115458.110205-105717464505158/source",
+                    "unsafe_writes": null,
+                    "validate": null
+                }
+
 msg
 ```
 A string with a generic message relayed to the user.
 
+    .. code-block:: console
+
+        "msg": "line added"
+
 rc
 ``
 Some modules execute command line utilities or are geared for executing commands directly (raw, shell, command, etc), this field contains 'return code' of these utilities.
 
+    .. code-block:: console
+
+        "rc": 257
+
 results
 ```````
 If this key exists, it indicates that a loop was present for the task and that it contains a list of the normal module 'result' per item.
 
+    .. code-block:: console
+
+        "results": [
+            {
+                "ansible_loop_var": "item",
+                "backup": "foo.txt.83170.2020-07-30@07:03:05~",
+                "changed": true,
+                "diff": [
+                    {
+                        "after": "",
+                        "after_header": "foo.txt (content)",
+                        "before": "",
+                        "before_header": "foo.txt (content)"
+                    },
+                    {
+                        "after_header": "foo.txt (file attributes)",
+                        "before_header": "foo.txt (file attributes)"
+                    }
+                ],
+                "failed": false,
+                "invocation": {
+                    "module_args": {
+                        "attributes": null,
+                        "backrefs": false,
+                        "backup": true,
+                        "content": null,
+                        "create": false,
+                        "delimiter": null,
+                        "directory_mode": null,
+                        "firstmatch": false,
+                        "follow": false,
+                        "force": null,
+                        "group": null,
+                        "insertafter": null,
+                        "insertbefore": null,
+                        "line": "foo",
+                        "mode": null,
+                        "owner": null,
+                        "path": "foo.txt",
+                        "regexp": null,
+                        "remote_src": null,
+                        "selevel": null,
+                        "serole": null,
+                        "setype": null,
+                        "seuser": null,
+                        "src": null,
+                        "state": "present",
+                        "unsafe_writes": null,
+                        "validate": null
```

Comment:
I think it is better to add a shorter invocation example

Reply:
Thank you for the feedback!